lib/client/api.go

```diff
@@ -1869,19 +1869,10 @@ func (tc *TeleportClient) ExecuteSCP(ctx context.Context, cmd scp.Command) (err
 	}
 	defer proxyClient.Close()
 
-	// which nodes are we executing this commands on?
-	nodeAddrs, err := tc.getTargetNodes(ctx, proxyClient)
-	if err != nil {
-		return trace.Wrap(err)
-	}
-	if len(nodeAddrs) == 0 {
-		return trace.BadParameter("no target host specified")
-	}
-
 	nodeClient, err := tc.ConnectToNode(
 		ctx,
 		proxyClient,
-		NodeDetails{Addr: nodeAddrs[0], Namespace: tc.Namespace, Cluster: tc.SiteName},
+		NodeDetails{Addr: serverAddr + ":0", Namespace: tc.Namespace, Cluster: tc.SiteName},
```

Comment:
Yeah I'm not sure what the `:0` is or why it's necessary. Once you figure it out, a comment is probably warranted.

Reply:
Added a comment for the "why" but not really the "WHY why"... just the why _I'm_ doing it. Will still dig into it